apps/settings/elements/firefox_accounts.html

```diff
@@ -29,14 +29,14 @@ <h2 data-l10n-id="fxa-confirm-account">Confirm Your Account</h2>
       </header>
       <ul>
         <li class="description">
-          <p data-l10n-id="fxa-verification-email-sent-msg" id="fxa-unverified-text">
+          <p id="fxa-unverified-text">
             We will send an email to: foo@bar.com
           </p>
           <p data-l10n-id="fxa-please-verify">
             Click the verification link in your email to confirm your account.
           </p>
-          <p id="fxa-resend-email">
-            Don't see an email? <a href="#">Resend</a>.
+          <p id="fxa-resend-email" data-l10n-id="fxa-dont-see-email-resend">
+            Don't see an email? <a href="#" id="fxa-resend">Resend</a>.
```

Comment:
I didn't quite get it when replying the comments above. So correct me if I'm wrong:
1. the new `fxa-dont-see-email-resend` string has a `<a>` tag so it will be applied by following the same DOM structure we defined here as
`Don't see an email? <a href="#" id="fxa-resend">Resend</a>` without removing `href` and `id` attributes, right? (I remember it's an awesome feature of l20n.)
2. what @gasolin suggested was to remove `Don't see an email?` and `Resend` text in HTML because it will be overwritten at build time anyway? If so, I prefer to keep it because it's not purely a string but includes a HTML tag. Keeping the content is more readable to me.